auth/src/main/java/io/camunda/zeebe/auth/api/TenantAccessChecker.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.auth.api;
+
+import java.util.List;
+
+public interface TenantAccessChecker {
```

Comment:
:+1: Makes sense. We have this inconsistency in C7, where the class is called `TenantCommandChecker` but we have `authorizedTenants`. It would be nicer if we align them in C8. I'll apply the changes.